xbmc/windowing/Resolution.cpp

```diff
@@ -105,7 +103,7 @@ void CResolutionUtils::FindResolutionFromWhitelist(float fps, int width, int hei
 
   CLog::Log(LOGDEBUG, "Trying to find exact refresh rate");
 
-  for (const auto& mode : indexList)
+  for (const auto &mode : indexList)
```

Comment:
why did you change this? we left align in Kodi :stuck_out_tongue_closed_eyes:

Reply:
In order to properly revert it. I will add it as a cosmetics commit